Cimpress.Stereotype/StereotypeRequest.cs

```diff
@@ -40,18 +41,29 @@ public IStereotypeRequest SetTemplateId(string templateId)
             _templateId = templateId;
             return this;
         }
+        
+        public IStereotypeRequest SetAcceptHeader(string acceptHeader)
```

Comment:
Would it be possible avoid the semantics of HTTP communication and use more abstract language of Stereotype. Something like `SetExpectedContent` or `SetExpectation(string mimeType)` something like that...?

Reply:
How would we model complex accept headers with probabilities?

Reply:
`SetExpectation(string mimeType, decimal preference)` ?

Reply:
Added